ReactiveCocoa/Swift/Observer.swift

```diff
@@ -56,11 +67,17 @@ public struct Observer<Value, Error: ErrorType> {
 
 extension Observer: ObserverType {
 	/// Puts a `Next` event into the given observer.
+	///
+	/// - parameters:
+    ///   - value: a value sent with the `Next` event
```

Comment:
Ha! 😄   So I'm cherry picking the commit you cooked up with the indentation changes which will fix this up.  At the same time I need to investigate why my Xc is being so weird with tabs and spaces, afaiknew it normally did conversions in place.